### PR TITLE
enhancement: next iteration of search result UI updates

### PR DESCRIPTION
**Pull request by schalkneethling**
Second iteration of the search results page UI updates

Part of #2889

### CODE REVIEW

**Comment by peterbe** [comment]
Why did you remove the gap between the title and the excerpts? It was better before. 
When it's like this it doesn't feel like the excerpts belong to the little. It reads like:
1. Title & URL
2. Excerpts
3. Title & URL
4. Excerpts
5. Title & URL
6. Excerpts

Before it was more like:

1. Title & URL & Excerpts
2. Title & URL & Excerpts
3. Title & URL & Excerpts

Screen Shot 2021-03-01 at 3 54 43 PM

**Comment by peterbe** [comment]
Perhaps this is off topic but why do the search results get to have an underline and the sort options don't?
Screen Shot 2021-03-01 at 4 02 03 PM

File: client/src/site-search/index.scss [hunk 1]
```diff
@@ -1,5 +1,14 @@
 .site-search {
   min-height: 500px;
+
+  a {
```
Comment (peterbe) [inline client/src/site-search/index.scss:4]:
```suggestion
  // This is temporary. 
  // Once https://github.com/mdn/yari/issues/2901 lands we can delete this. 
  a {
```

**Review by peterbe — APPROVED** [review]
Wonderful! 
But a nit so that we can't forget to undo the hack to underlinging `a`s.

**Comment by peterbe** [comment]
How did you manage to merge it when the CI checks didn't pass? I thought the merge button would be disabled.

**Comment by schalkneethling** [comment]
> How did you manage to merge it when the CI checks didn't pass? I thought the merge button would be disabled.

🤷‍♂️  No idea, it was green when I clicked it.

**Comment by peterbe** [comment]
> > How did you manage to merge it when the CI checks didn't pass? I thought the merge button would be disabled.
> 
> 🤷‍♂️ No idea, it was green when I clicked it.

That makes sense. It's s really complex machinery and GitHub is constantly working on it so it's very possibly a race condition or a UI bug.

**Pull request merged**